tasks/opensearch.yml

```diff
@@ -35,3 +35,14 @@
   loop:
     - Xms
     - Xmx
+
+- name: Install OpenSearch analysis-icu plugin
+  notify: Restart opensearch
+  ansible.builtin.command:
+    argv:
+      - /usr/share/opensearch/bin/opensearch-plugin
+      - install
+      - --silent
+      - analysis-icu
+    creates: /usr/share/opensearch/plugins/analysis-icu
+  when: opencast_version_major | default(16) | int >= 17
```

Comment:
Don't do this. Instead, in line 4, install `opensearch-plugin-analysis-icu`

Reply:
Will update this PR

Reply:
There are no opensearch plugins in Debian repo :(

```
apt search opensearch
Sorting... Done
Full Text Search... Done
libwww-opensearch-perl/stable 0.17-2 all
  search OpenSearch compatible web sites

opensearch/17.x 1.3.16 amd64
  An open source distributed and RESTful search engine

opensearch-dist/17.x 1.3.3-1 amd64
  OpenSearch is a community-driven, Apache 2.0-licensed open source
```

Reply:
Maybe it's still in the testing repo? AFAIK Greg wanted to package it. Let's ask him.

Reply:
Talked to Greg. He said the Debian packages should be available later today or tomorrow.